tests/conftest.py

```diff
@@ -354,6 +356,36 @@ def finalizer():
     return factory
 
 
+@pytest.fixture()
+def teardown_factory(request):
+    """
+    Tearing down a resource that was created during the test
```

Comment:
Can we add more docstring here on where/how to use this? I am not clear on the usecase

Reply:
done.

Please let me know if I need to add more